lms/djangoapps/courseware/permissions.py

```diff
@@ -5,4 +5,8 @@
 from bridgekeeper import perms
 from .rules import HasAccessRule
 
-perms['courseware.view_course_home'] = HasAccessRule('load')
+VIEW_COURSE_HOME = 'courseware.view_course_home'
+VIEW_COURSEWARE = 'courseware.view_courseware'
```

Comment:
I did a similar thing in the PR I'm working on (that defines `HasAccessRule`). I named mine `PERM_VIEW_COURSE_HOME` to make it clear after imports that it was a permission... But maybe just the fact that it's being imported from the `permissions` model is enough?

Otherwise, yes, this looks good.